src/test/java/duke/misc/ParserTest.java

```diff
@@ -0,0 +1,23 @@
+package duke.misc;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import duke.exception.DukeException;
+import duke.task.Event;
+
+class ParserTest {
+    @Test
+    public void testMakeTask() {
```

Comment:
Could perhaps name the test method in a way thats more descriptive of the nature and the SUT?
e.g. `featureUnderTest_testScenario_expectedBehavior()`